tests/framework/integration/cluster.go

```diff
@@ -790,8 +802,8 @@ func (m *Member) clientScheme() string {
 
 func (m *Member) addBridge() (*bridge, error) {
 	network, host, port := m.grpcAddr()
-	grpcAddr := host + ":" + port
-	bridgeAddr := grpcAddr + "0"
+	grpcAddr := net.JoinHostPort(host, m.Port)
+	bridgeAddr := net.JoinHostPort(host, fmt.Sprintf("%s%s", port, "0"))
```

Comment:
Should bridge also bind a random port instead of basing it of `m.Port`?

Reply:
That's a good idea, not sure that append 0 trick works all the time - I've left it for the `unix://` for the time being.